requirements/requirements.in

```diff
@@ -28,6 +28,7 @@ jinja2
 jsonschema
 Markdown  # used for formatting API help
 openshift
+pexpect==4.7.0 # see library notes
```

Comment:
I don't see any notes for this. I would, myself, like some kind of documented reason why this is pinned to a specific version.

Reply:
forgot to git add it. Good eye.